README.md

```diff
@@ -366,7 +302,8 @@ influence the roadmap as follows:
   board](https://zenml.io/discussion). You can also request for new features here.
 - Start a thread in our [Slack channel](https://zenml.io/slack-invite).
```

Comment:
Maybe also mention "create github issue" here? @strickvl or is that even a recommended way of requesting features? I remember we discussed at some point whether all feature requests should go over the roadmap instead, what was the decision on this again?